spec/yaml/swiftnav/sbp/piksi.yaml

```diff
@@ -9,133 +9,101 @@
 # WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
 
 package: swiftnav.sbp.piksi
-description: System messages from the Piksi.
+description: |
+  System health, configuration, and diagnostic messages specific to
+  the Piksi L1 receiver, including a variety of legacy messages that
+  may no longer be used. These messages are in the
+  implementation-defined range (0x0000-0x00FF), and largely intended
+  for internal-use only.
 stable: False
 public: True
 include:
   - types.yaml
 definitions:
 
- - MSG_PRINT:
-    id: 0x0010
-    public: True
-    desc: Information and debugging information.
-
- - MSG_DEBUG_VAR:
-    id: 0x0011
-    desc: Legacy message for tracing variable values.
-
  - MSG_ALMANAC:
     id: 0x0069
-    desc: MSG_ALMANAC
+    short_desc: Legacy message to load satellite almanac (Host => Piksi).
+    desc: |
+      This is a legacy message for sending and loading a satellite
+      alamanac onto the Piksi's flash memory from the host.
 
  - MSG_SET_TIME:
     id: 0x0068
-    desc: MSG_SET_TIME
-
- - MSG_BOOTLOADER_HANDSHAKE:
-    id: 0x00B0
-    desc: MSG_BOOTLOADER_HANDSHAKE
-
- - MSG_BOOTLOADER_JUMP_TO_APP:
-    id: 0x00B1
-    desc: MSG_BOOTLOADER_JUMP_TO_APP
+    short_desc: Send GPS time from host (Host => Piksi).
+    desc: |
+      This message sets up timing functionality using a coarse GPS
+      time estimate sent by the host.
 
  - MSG_RESET:
     id: 0x00B2
-    desc: Reset the devices.
+    short_desc: Reset the device (Host => Piksi).
+    desc: |
+      This message from the host resets the Piksi back into the
+      bootloader. It ensures that all outstanding memory accesses
+      including buffered writes are completed before reset begins.
 
  - MSG_CW_RESULTS:
```

Comment:
We probably want to remove these CW messages altogether.  The current FPGA has the CW channel removed.